README.md

```diff
@@ -228,6 +228,30 @@ client.proxy_url('pod', 'podname', 5001, 'ns')
 ```
 
 
+#### Get the logs of a pod
+You can get the logs of a running pod:
+
+```ruby
+client.get_pod_log('pod-name', 'default')
+ => "Running...\nRunning...\nRunning...\n"
+```
+
+If that pod has more than one container, you must specify the container:
+
+```ruby
+client.get_pod_log('pod-name', 'default', container: 'ruby')
```

Comment:
I haven't found documentation in the API that states it is necessary. But if I try running `kubectl logs` on a pod that has more than one container, I get this error message:

```
$ kubectl logs account-service-rc-ci2bc
error: POD account-service-rc-ci2bc has more than one container; please specify the container to print logs for
```